src/test/java/codeu/controller/ConversationServletTest.java

```diff
@@ -161,10 +164,13 @@ public void testDoPost_ConversationNameTaken() throws IOException, ServletExcept
   @Test
   public void testDoPost_NewConversation() throws IOException, ServletException {
     Mockito.when(mockRequest.getParameter("conversationTitle")).thenReturn("test_conversation");
+    Mockito.when(mockRequest.getParameter("conversationUser")).thenReturn("test_user_added");
     Mockito.when(mockSession.getAttribute("user")).thenReturn("test_username");
 
     User fakeUser = new User(UUID.randomUUID(), "test_username", "test_username", Instant.now());
+    User fakeUserAdded = new User(UUID.randomUUID(), "test_user_added", "test_user_added", Instant.now());
```

Comment:
This code is repeated in a couple places, it could be moved to the init. Also, for the second string, instead of "test_user_added", it would be clearer to use "password" (even though that's not how passwords are actually stored, it makes it clearer what the field is for).

Reply:
do you mean move it to the setup() part?

Reply:
Yup!

Reply:
Done :)